docs/DeveloperGuide.md

```diff
@@ -602,29 +602,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a … | I want to … | So that I can…|
 | - | - | - | - |
-| `* * *` | new user | see usage instructions | refer to instructions if I forget how to use the app |
 | `* * *` | confused user | have a help command | learn how to use the application |
-| `* * *` | user | add a new person | |
-| `* * *` | user | add issues | |
-| `* * *` | user | see a list of pending tasks sorted by deadline | prioritise which to do |
-| `* * *` | user | search for tasks | |
+| `* * *` | user | add issues | track associated issues with rooms and their status|
+| `* * *` | user | see a list of pending issues sorted by deadline | prioritise which issue to work on |
+| `* * *` | user | search for issues | find a specific issue and get its information |
+| `* * *` | user | search for issues | find issues matching a specific criteria |
```

Comment:
Hmm,

> search for issues

seems to be repeated.

Reply:
This was intentional. I wanted to show that you can either find 1 issue (for instance for room 10-111) or all rooms with cockroaches (matching a specific criteria). What do you think?

Reply:
Oh, then maybe distinguish them as
"search for issues by room number" and
"search for issues by description"

Reply:
Alright